.travis.yml

```diff
@@ -9,3 +9,12 @@ script:
  - dotnet restore
  - dotnet build -c Release
  - dotnet test Functional.Primitives.FluentAssertions.Tests/Functional.Primitives.FluentAssertions.Tests.csproj
+ - cd ../../src/Functional.Unions.FluentAssertions
```

Comment:
As some housekeeping, I may just migrate everything into a single solution so that the build pipeline is simplified.  I'm not too worried about it since this repository is rarely updated.